packages/http-client-csharp/generator/Microsoft.Generator.CSharp.TestPlugin/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Generator.CSharp.ClientModel.Plugin.Tests")]
```

Comment:
lets only give internals visibile to our own test project.  Each plugin should have its own it looks like we are crossing projects here?